### PR TITLE
fix(LogoGrid): pattern stretch to full screen

### PR DESCRIPTION
**Pull request by reesedownes**
### Related Ticket(s)

Pattern: LogoGrid - The card must be stretched to full screen size as per specs. #2604

### Description

logos now stretch to fill the screen on all breakpoints according to design specs

### Changelog

**Changed**

- logo-grid.scss has several fixes

### CODE REVIEW

**Comment by reesedownes** [comment]
@guilhermelMoraes no worries, man. good catch; i thought that CTA wasn't supposed to "hang" in the margins until the 3-card layout, but i was wrong. you also made me realize i was missing a breakpoint, so thanks.

File: packages/styles/scss/patterns/blocks/logo-grid/_logo-grid.scss [hunk 1]
```diff
@@ -27,49 +25,51 @@
...
         @include carbon--type-style('productive-heading-01');
       }
     }
 
-    a.#{$prefix}--card__CTA {
-      max-width: 100%;
-      margin-left: 0;
-    }
-
     &__link {
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/logo-grid/_logo-grid.scss:49]: Starting here started outside of your component class, so you may want to start looking here.

File: packages/styles/scss/patterns/blocks/logo-grid/_logo-grid.scss [hunk 2]
```diff
@@ -124,6 +125,40 @@
       }
     }
   }
+
+  .#{$prefix}--content-block__cta-row {
```
Comment (jeffchew) [inline packages/styles/scss/patterns/blocks/logo-grid/_logo-grid.scss:129]: This whole block is outside of the component class.

**Comment by reesedownes** [comment]
@kennylam just so i understand, are you saying the changes that i made to the CTA card across the board? or just in `.#{$prefix}--card__CTA` specifically?

**Comment by kennylam** [comment]
> just so i understand, are you saying the changes that i made to the CTA card across the board? or just in `.#{$prefix}--card__CTA` specifically?

Right, the change I suggested would fix CTA across the board in `ContentBlock`, and some of the patterns using block are still off, I believe.

**Comment by jeffchew** [comment]
Closing this, @kennylam will be opening a new PR.

**Pull request closed without merging**